internetarchive/session.py

```diff
@@ -287,14 +292,15 @@ def get_metadata(self, identifier, request_kwargs=None):
             raise type(exc)(error_msg)
         return resp.json()
 
-    def search_items(self, query,
-                     fields=None,
-                     sorts=None,
-                     params=None,
-                     full_text_search=None,
-                     dsl_fts=None,
-                     request_kwargs=None,
-                     max_retries=None):
+    def search_items(self,
+                     query: str,
+                     fields: list | None = None,
+                     sorts: Any | None = None,
```

Comment:
I think these should be `Iterable[str]`, namely iterables of field names. They're normally lists, but I don't see why tuples or other iterables wouldn't work just as well.

Reply:
Thanks.